packages/react/src/button/Button.test.tsx

```diff
@@ -84,6 +86,50 @@ describe('Button', () => {
 			expect(liveRegion).toHaveTextContent('Busy');
 		});
 	});
+
+	describe('With Icon', () => {
```

Comment:
Added some tests here since we didn't have coverage for small buttons or buttons with icons